Cubical/Foundations/Isomorphism.agda

```diff
@@ -125,6 +125,11 @@ rightInv (compIso (iso _ _ rightInv _) (iso g ginv rightInv' _)) b =
 leftInv (compIso (iso f finv _ leftInv) (iso _ _ _ leftInv')) a =
   cong finv (leftInv' (f a)) ∙ leftInv a
 
+infixr 33 _⋄_
+
+_⋄_ : _
+_⋄_ = compIso
```

Comment:
I'm not sure I like this notation... Can we please add this in a separate PR? It should change most uses of compIso as well...

Reply:
hehe yeah, I just improvised since Guillaume thought it would be appropriate to have something like this for my massive compositions. Is it better if I just import compIso as ⋄ (or perhaps some more appropriate symbol better) in my modules? The problem then is that I have to add infixr everytime, right? Alternatively, I can add infixr to compIso.

Reply:
Oh wait, I don't think I even use this hehe... I could just remove it if you want.

Reply:
Yeah, let's just remove it and have a separate discussion about notations for compIso and compEquiv. If we introduce an infix operation for these we should use them consistently throughout the library. So if we decide to do this it should go into a separate PR doing only that